store/mongo/migration_1_7_0.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package mongo
+
+import (
+	"context"
+	cinv "github.com/mendersoftware/deviceauth/client/inventory"
+	dconfig "github.com/mendersoftware/deviceauth/config"
+	"time"
+
+	"github.com/mendersoftware/go-lib-micro/config"
+	"github.com/mendersoftware/go-lib-micro/identity"
+	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
+	ctxstore "github.com/mendersoftware/go-lib-micro/store"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/mendersoftware/deviceauth/model"
+)
+
+type migration_1_7_0 struct {
+	ms  *DataStoreMongo
+	ctx context.Context
+}
+
+const (
+	migrationContextTimeouts = 360
```

Comment:
360 seconds to set up to 512 device statuses? Maybe we can lower it, I believe 60 seconds are already too much!

Reply:
sure.-> 32 the way you like it :)